esp/services/ws_packageprocess/ws_packageprocessService.cpp

```diff
@@ -330,7 +330,7 @@ class PackageMapUpdater
 #else
```

Comment:
Did you mean to remove the #ifdef?

Reply:
Yes, it should be removed. The problem happened when I rebased the code to 8.2.0. I found the same problem in another place.